### ISSUE TITLE
decide on communication channels for development team

### ISSUE THREAD

**Issue by vkuehne** [hospitalityclub/hc-django#2]
As more developers show interest (awesome!), let's decide how to work together. From what I understand we need:
- tracker: shall we just use the github-built in, or something like trello?
- chat: slack is the channel of choice for you guys? Or anything else?

What else do we need, for documentation the tools here are enough?

**Comment by vkuehne** [comment]
Ok, so @infohata and @infrarosso78 agree that github tracker is fine for us, so guess that's settled.

What's next? :)

**Comment by infohata** [comment]
Yes. Projects need to be set up to sort issues, or we can just use labels.

**Comment by infohata** [comment]
please put your telegram usernames here. Mine is @infohata 
I'll create a hc dev telegram group and make Veit the owner =)

**Comment by alvarotor** [comment]
Hi everybody, tracking issues publicly here in github is great. For ticketing trello is a great tool and for communication I'd def vote for slack, but whatsapp or telegram groups can also suffice for most needs.
I'd say that most small groups mostly work in that way.

**Comment by vkuehne** [comment]
Telegram @vkuehne (just had to create a new account, bc mine apparently was self-destroyed bc I didn't use it for so long)

**Comment by vkuehne** [comment]
Just had a look at trello again - don't quite get the difference between ticketing there, and just using the tracker here. But up to you guys - I'm ready for anything :)

**Comment by infohata** [comment]
I'd not take Trello, it is a commercial product with insane limitations for unpaid version. Of course, it is nice to have a visual board but from my practice, it gets out of visually workable appearance pretty quickly - you can manage 10s of issues on board, but not 1000s as we are potentially looking at.

In terms of dev team chat, it's on [Telegram](https://t.me/hc4dev).

**Comment by alvarotor** [comment]
> I'd not take Trello, it is a commercial product with insane limitations for unpaid version. Of course, it is nice to have a visual board but from my practice, it gets out of visually workable appearance pretty quickly - you can manage 10s of issues on board, but not 1000s as we are potentially looking at.
> 
> In terms of dev team chat, it's on [Telegram](https://t.me/hc4dev).

Yes, its a good point. Specially thinking that is a commercial product. But its a good tool anyway to track issues. Doing it here has the advantage that is public.
